File: tests/fixtures/generic/vulnerable/xss_raw_html.cs
// @audit-fixture
// @rule: xss_raw_html
// @category: security
// @expected: detected
// VULNERABLE: rendu HTML brut via @Html.Raw depuis input HTTP (CWE-79)
public class PageController {
    public string Render() {
        var userInput = Request.QueryString["name"];
        return Html.Raw(userInput);
    }
}

File: tests/fixtures/generic/vulnerable/xss_raw_html.py
# @audit-fixture
# @rule: xss_raw_html
# @category: security
# @expected: detected
# VULNERABLE: rendu HTML brut d'une entrée utilisateur (CWE-79)
from flask import request

def render():
    user_input = request.args.get("name", "")
    return Html.Raw(user_input)
